cfme/tests/containers/test_labels.py

```diff
@@ -0,0 +1,230 @@
+import json
+import random
+import time
+
+import fauxfactory
+
+from itertools import chain
+
+import pytest
+
+import requests
+
+from cfme.containers.image import Image
+from cfme.containers.node import Node
+from cfme.containers.pod import Pod
+from cfme.containers.project import Project
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.replicator import Replicator
+from cfme.containers.service import Service
+from cfme.containers.template import Template
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable, paginator, toolbar as tb
+
+from utils.log import logger
+
+from utils import testgen, version
+from utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: version.current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    [ContainersProvider], scope='function')
+
+
+TEST_OBJECTS = [Project, Service, Replicator, Pod, Node, Template, Image]
+
+objects_ocp = ['"Namespace"', '"Service"', '"ReplicationController"', '"Pod"',
+               '"Node"', '"Template"', '"Image"']
+
+labels_for_objs = ['project', 'service', 'rc', 'pod', 'node', 'template', 'image']
+
+
+@pytest.fixture(scope="module")
+def strip_string(found_item):
+    a_string = str(found_item)
+    a_exists = a_string[1:-1]
+    a_item = a_exists[1:-1]
+
+    return a_item
```

Comment:
Why is this a fixture?
Why not just:
def strip_string(found_item):
    return (found_item[2:-2])